hikvision_tftpd.py

```diff
@@ -55,10 +55,10 @@ def _bind(self, addr):
                 raise Error(
                     ('Address %s:%d not available.\n\n'
                      'Try running:\n'
-                     'linux$ sudo ifconfig eth0:0 %s\n'
+                     'linux$ sudo ip addr add 192.0.0.128/24 dev enp4s0 %s\n'
                      'osx$   sudo ifconfig en0 alias %s '
                      '255.255.255.0\n\n'
-                     '(adjust eth0 or en0 to taste. see "ifconfig -a" output)')
+                     '(adjust eth0 or en0 to taste. see "ip addr" output)')
```

Comment:
If you change the eth0 to enp4s0 above, please adjust it here too so the message is self-consistent